FlorinPosea/projects/react-tic-tac-toe/src/components/cell-board/CellBoard.css

```diff
@@ -0,0 +1,90 @@
+.cell {
+  width: var(--cell-size);
+  height: var(--cell-size);
+  border: 1px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  cursor: pointer;
+}
+
+.cell:first-child,
+.cell:nth-child(2),
+.cell:nth-child(3) {
+  border-top: none;
+}
+
+.cell:nth-child(3n + 1) {
+  border-left: none;
+}
+
+.cell:nth-child(3n + 3) {
+  border-right: none;
+}
+
+.cell:last-child,
+.cell:nth-child(8),
+.cell:nth-child(7) {
+  border-bottom: none;
+}
+
+.cell.x,
+.cell.circle {
+  cursor: not-allowed;
```

Comment:
In exemplul de javascript simplu, cand nu e ceva in cell, (nu e X, nu e O), atunci cursor-ul e pointer. Si daca e ocupat, atunci "not-allowed"
![image](https://user-images.githubusercontent.com/45266485/201580936-809db814-c1f3-4073-8443-804bca81b7a7.png)